src/main/java/bio/terra/service/filedata/azure/util/BlobContainerCopier.java

```diff
@@ -0,0 +1,222 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobUrlParts;
+import com.azure.storage.blob.models.BlobCopyInfo;
+import com.azure.storage.blob.models.BlobListDetails;
+import com.azure.storage.blob.models.ListBlobsOptions;
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BlobContainerCopier {
+
+  private static final int MIN_LIST_OPERATION_TIMEOUT_IN_SECONDS = 10;
+  private static final int MIN_POLLING_INTERVAL_IN_SECONDS = 2;
```

Comment:
It looks like these are only ever used as durations. If so, you could declare as `Duration` and then you'd no loner need `_IN_SECONDS` in the names.
```suggestion
  private static final Duration MIN_LIST_OPERATION_TIMEOUT = Duration.ofSeconds(10);
  private static final Duration MIN_POLLING_INTERVAL = Duration.ofSeconds(2);
```